### ISSUE TITLE
[Accessibility]: Cannot navigate to popover buttons using keyboard

### ISSUE THREAD

**Issue by varungandhi-src** [sourcegraph/sourcegraph#36567]
### Audit type

Keyboard navigation

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/33509

### Problem description

After the code intel popover has activated, it doesn't seem like it is possible to navigate to the buttons like Go to Definition, Find References and Find Implementations using only the keyboard.

(Putting this under "Go to Definition" but it also applies to "Find References" and related user journeys.)

### Expected behavior

It should be possible to navigate to the buttons and "click" them using the keyboard.

### Assigning labels

- [ ] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [ ] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [ ] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.

**Comment by umpox** [comment]
The primary reason for this is that we cannot tab directly to a token and thus can't then tab into the popover.

We have plans to fix this in the near future, that will also close this ticket.

Closing this in favour of:
- https://github.com/sourcegraph/sourcegraph/issues/42420
- https://github.com/sourcegraph/sourcegraph/issues/42421
